code/_expts/slotnick_etal_2001/analyze/plots/de_CreateSlotnickFigure1.m

```diff
@@ -4,25 +4,25 @@
 %  E1:  errorbar e
 %  taskTitle: slotnick task (captialized first letter)
 
-if ~exist('ax', 'var')
-    % Create figure
-    figure1 = de_NewFig(sprintf('%s_%s', stimSet, lower(taskTitle)));
-    ax = gca(); % make current figure the axis if not passed in.
-end
+    if ~exist('ax', 'var')
+        % Create figure
+        figure1 = de_NewFig(sprintf('%s_%s', stimSet, lower(taskTitle)));
+        ax = gca(); % make current figure the axis if not passed in.
+    end
```

Comment:
@vishaalprasad Note that I indent all code within a function, and never use `end` to end a function. Just style consistency things.